prime-angular-frontend/src/app/modules/adjudication/shared/components/health-authority-table/health-authority-table.component.html

```diff
@@ -10,14 +10,22 @@
             *matHeaderCellDef
             scope="col">&nbsp;</th>
         <td mat-cell
-            *matCellDef="let row;"
-            class="prefix">
+            *matCellDef="let row;">
+
           <button *ngIf="isHealthAuthority(row) && flaggedHealthAuthorities?.includes(row.id)"
                   mat-icon-button
                   title="View Pending Authorized Users"
                   (click)="onRoute([AdjudicationRoutes.HEALTH_AUTHORITIES, row.id, AdjudicationRoutes.HEALTH_AUTH_AUTHORIZED_USERS])">
             <mat-icon class="red-icon">admin_panel_settings</mat-icon>
           </button>
+
+          <div *ngIf="!isHealthAuthority(row)"
+               class="d-flex flex-wrap prefix-container">
+            <app-contextual-site-notification *ngIf="row.hasNotification"
```

Comment:
Noticed that if I remove notice it performs the action properly, but the red notification icon doesn't get removed until you refresh the page.

Reply:
looks like i missed watching the reload event!